index.html

```diff
@@ -54,37 +54,38 @@
                 <li><img src="./src/hamburger-logos/partners.png" alt="founders-btn">
                     <a href="#partners" class="menu-icons">Partners</a></li>
                 <li><img src="./src/hamburger-logos/about.png" alt="about-btn">
-                    <a href="#featured-speakers" class="menu-icons">About</a></li>
+                    <a href="./about.html" class="menu-icons">About</a></li>
             </ul>
         </div>
     </div>
     <main class="main-website-container">
         <section id="home-page">
 
         <div class="titles">
-            <h2 class="main-titling">The first online school to invest in you.</h2>
+            <span class="greetings">"Hello world!"</span>
+            <h1 class="main-titling">THE FIRST ONLINE SCHOOL TO INVEST IN YOU.</h1>
         </div>
 
         <div class="main-description">
             <p>We’re experts at training people with zero coding experience and turning them into high-earning web developer engineers. We've helped over 6000 students get hired at over 3,000 companies. Join now and become the web developper you've always wanted to be</p>
             <div class="due-date">
-                <h1>2021.10.15 (Wednesday)</h1>
-                <h2>Join before the next due date</h2>
+                <h3>2021.10.15 (Wednesday)</h3>
+                <h4>Join before the next due date</h4>
```

Comment:
- [ ] The greetings `Hello world!` doesn't appear on the screen. 

- [ ] Let us include first paragraph in the card as we have in the original design and modify the font-size of our dates.

Yours

![capstone5](https://user-images.githubusercontent.com/55635977/132565244-829d95af-51de-4b4d-a2e1-e4ad77fe6b7f.png)

Original

![capstone](https://user-images.githubusercontent.com/55635977/132564358-5d8cd889-ff13-44aa-b686-334e08603edc.png)

- [ ] Let us align our First Heading, paragraph and h3 with dates to start on the same line, check the image below

Yours

![capstone5](https://user-images.githubusercontent.com/55635977/132565244-829d95af-51de-4b4d-a2e1-e4ad77fe6b7f.png)

Original

![capstone1](https://user-images.githubusercontent.com/55635977/132564363-839083e7-ccf3-4726-8a74-a6024859c2b3.png)

- [ ] Let us add the footer section. 
 
![capstone3](https://user-images.githubusercontent.com/55635977/132564372-b7f103e1-5f44-46c5-87bb-aae5924c902a.png)

Reply:
@Emmyn5600 the changes you have requested are already there man. They probably just did not load on the live demo for some reasons but they are already. Excluding the footer which is not present but I'll add it right away